packages/contracts-bedrock/contracts/universal/CrossDomainMessenger.sol

```diff
@@ -263,15 +263,23 @@ abstract contract CrossDomainMessenger is
         bytes calldata _message
     ) external payable nonReentrant whenNotPaused {
         (, uint16 version) = Encoding.decodeVersionedNonce(_nonce);
-
-        // Block any messages that aren't version 1. All version 0 messages have been guaranteed to
-        // be relayed OR have been migrated to version 1 messages. Version 0 messages do not commit
-        // to the value or minGasLimit fields, which can create unexpected issues for end-users.
         require(
-            version == 1,
-            "CrossDomainMessenger: only version 1 messages are supported after the Bedrock upgrade"
+            version == 0 || version == 1,
+            "CrossDomainMessenger: only version 0 or 1 messages are supported"
```

Comment:
This could also be `version < 3`, its a `uint` so no need to worry about negative values

Reply:
> Think you mean `version < 2`.

Yes this is what i meant here